Cargo.toml

```diff
@@ -91,7 +91,7 @@ cop_datatype = { path = "components/cop_datatype" }
 git = "https://github.com/pingcap/murmur3.git"
 
 [dependencies.rocksdb]
-git = "https://github.com/pingcap/rust-rocksdb.git"
+git = "https://github.com/solotzg/rust-rocksdb.git"
```

Comment:
Please update the repo.

Reply:
However, there are lots of conflictions after this PR [rust-rocksdb 230](https://github.com/pingcap/rust-rocksdb/pull/230)

Reply:
What do you mean?

Reply:
Mainly because of the version of `libc`.